src/madhuraMJun20/scannerClass/DuplicateNumInArray.java

```diff
@@ -0,0 +1,19 @@
+/*Find duplicate number on Integer array in Java?*/
+package madhuraMJun20.scannerClass;
+
+public class DuplicateNumInArray {
+	void findDuplicate(int[] num) {
+		for (int i = 0; i < num.length; i++) {
+			for (int j = i + 1; j < num.length; j++) {
+				if (num[i] == num[j])
+					System.out.println(num[i] + " is a duplicate number");
```

Comment:
Variable name should be index and innerIndex instead of i and j.

Reply:
Done these changes.